planet/clients/data.py

```diff
@@ -175,11 +175,12 @@ async def create_search(self,
         url = self._searches_url()
 
         # TODO: validate item_types
-        request_json = {
+        request_json: typing.Dict[str, typing.Any] = {
             'name': name, 'filter': search_filter, 'item_types': item_types
         }
+        # QUESTION: can we just set the value to True or False above?
```

Comment:
We have a test that doesn't expect `__daily_email_enabled: False`, but I wonder if the API would handle this?